internal/pkg/source/entry.go

```diff
@@ -117,6 +118,37 @@ func (entries LazyLogEntries) Filter(term string) (LazyLogEntries, error) {
 	}, nil
 }
 
+// FilterRegEx filters entries by regex ignoring case
+func (entries LazyLogEntries) FilterRegExp(regex string) (LazyLogEntries, error) {
+	if regex == "" {
+		return entries, nil
+	}
+
+	re, err := regexp.Compile(strings.ToLower(regex))
```

Comment:
note: there’s a `(?i)` flag for case-insensitive matching, you just need to add it to the start of the regex if it isn't already there.

`(?i)[a-z0-9]+` will match both `AAAAA` and `aaaaaa`